randovania/game_connection/connector/dread_remote_connector.py

```diff
@@ -0,0 +1,85 @@
+import json
+import logging
+
+from randovania.game_connection.connector.remote_connector import PlayerLocationEvent, RemoteConnector
+from randovania.game_connection.executor.dread_executor import DreadExecutor
+from randovania.game_description import default_database
+from randovania.game_description.resources.item_resource_info import InventoryItem
+from randovania.game_description.db.region import Region
+from randovania.games.game import RandovaniaGame
+
+class DreadRemoteConnector(RemoteConnector):
+    _game_enum: RandovaniaGame = RandovaniaGame.METROID_DREAD
+
+    def __init__(self, executor: DreadExecutor):
+        super().__init__()
+        self._layout_uuid = executor.layout_uuid
```

Comment:
is this a string or the `uuid.UUID` type?

Reply:
Oops, yeah, that's a string